mmdet/core/mask/structures.py

```diff
@@ -297,6 +298,27 @@ def expand(self, expanded_h, expanded_w, top, left):
                           left:left + self.width] = self.masks
         return BitmapMasks(expanded_mask, expanded_h, expanded_w)
 
+    def shear(self,
+              shear_matrix,
+              out_shape,
+              fill_val=0,
+              flags=cv2.INTER_NEAREST,
```

Comment:
1. We should hide the cv2 options inside mmcv, see [here](https://github.com/open-mmlab/mmcv/blob/master/mmcv/image/geometric.py#L29).
2. For shear transformation, we need to first implement one operator in mmcv, which warps all the operations and cv2 options. Then call `mmcv.imgshear` in mmdet for all the places.

Reply:
Hi @ZwwWayne, so a PR which implements mmcv.imshear is required before this PR?